tests/builder.test.js

```diff
@@ -31,7 +32,7 @@ tap.test('Flow Manager Client Builder', async t => {
     brokers: conf.KAFKA_BROKERS_LIST,
   }
   const consumerConf = {
-    groupId: conf.KAFKA_GROUP_ID,
+    groupId: `${conf.KAFKA_GROUP_ID}_${crypto.randomBytes(12).toString('hex')}`,
```

Comment:
why the random groupId for these tests?

Reply:
This should be part of the changes @silversoul93 made in order to run the tests in parallel without problems.

Reply:
@fredmaggiowski as @SimoneBergonzi said, I added the random part to run tests in parallel without problems.